sycl/test/on-device/extensions/matrix-amx-int8-test.cpp

```diff
@@ -0,0 +1,171 @@
+// RUN: %clangxx -mamx-bf16 -mamx-int8 -mavx512bw -mavx512vbmi -fsycl -O2 %s -o %t.out
```

Comment:
Using -march=sapphirerapids instead of "-mamx-bf16 -mamx-int8 -mavx512bw -mavx512vbmi"?